packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/index.scss

```diff
@@ -0,0 +1,365 @@
+        z-index: 10;
+        width: 100%;
+
+        .dc-mobile-full-page-modal {
+            @include mobile {
+                width: 100%;
+            }
+
+            &__body {
+                height: 100%;
+            }
+        }
+
+        & .dc-tabs {
+            @include mobile {
+                height: 100%;
+                display: unset;
+                flex: 1;
+
+                &__content {
+                    height: calc(100% - 40px);
+                }
+            }
+        }
+    }
+
+    &__container {
+        @include desktop {
+            height: calc(80vh - 0.8rem - 4rem - 4.8rem - 5.8rem);
+
+            &--has-footer {
+                height: calc(80vh - 0.8rem - 4rem - 4.8rem - 5.8rem - 7.4rem);
+            }
+
+            > * {
+                height: 100%;
+                margin-top: 0.8rem;
+            }
+        }
+
+        @include tablet {
+            height: calc(80vh - 0.8rem - 4rem - 4.8rem - 5.8rem);
+
+            &--has-footer {
+                height: calc(80vh - 0.8rem - 4rem - 4.8rem - 5.8rem - 7.4rem);
+            }
+
+            > * {
+                height: 100%;
+                margin-top: 0.8rem;
+            }
+        }
+
+        @include mobile {
+            height: 100%;
+        }
+    }
+
+    &__title {
+        font-size: var(--text-size-s);
+        font-weight: 700;
+        margin: 1.5rem;
+    }
+
+    &__preview-workspace {
+        padding: 1.5rem;
+        border-radius: $BORDER_RADIUS;
+        border: solid 1px var(--border-normal);
+        height: calc(100% - 30rem); // TODO: What is 37px?
+        position: relative;
+        overflow: hidden;
+        margin-bottom: 1.6rem;
+        margin-right: 1.6rem;
+
+        &-controls {
+            padding: 0.7rem 0.5rem;
+            display: flex;
+            flex-direction: column;
+            position: absolute;
+            right: 1.6rem;
+            bottom: 1.6rem;
+            border-radius: 3rem;
+            background-color: $color-grey-2;
+            box-shadow: 0.2rem 0.2rem 0.5rem var(--shadow-menu);
+            z-index: 99;
+        }
+
+        &-icon {
+            margin: 0.5rem;
+            cursor: pointer;
+        }
+    }
+
+    &__recent {
+        display: flex;
+        flex-direction: row;
+        gap: 1.6rem;
+
+        &-files {
+            flex-basis: 100%;
+
+            &-list {
+                overflow: auto;
+                /* TODO: ThemedScrollbars! */
+                height: calc(100% - 52px); // 100% - Title height
+            }
+        }
+
+        &-preview {
+            flex-basis: 65%;
+            display: flex;
+            flex-direction: column;
+
+            &-title {
+                margin-left: 0;
+            }
+        }
+
+        &-empty {
+            display: flex;
+            align-items: center;
+            flex-direction: column;
+            justify-content: center;
+            text-align: center;
+
+            &-icon {
```

Comment:
all of these elements should be changed to __

Reply:
this is a legacy change will fix it in improvement PR because the older load model css is breaking